cc_utilities/legacy_upload.py

```diff
@@ -215,24 +216,6 @@ def generate_cc_dummy_patient_cases(
     upload_data_to_commcare(
         dummies_data, project_slug, "patient", "case_id", cc_user_name, cc_api_key
     )
-    # Because of an oddity of the CommCare API, even if cases are created with
-    # `close: "yes"`, the cases will appear as open in the case list dashboard.
-    # We therefore send `close_data` below to close them
-    close_data = [
-        dict(external_id=item["external_id"], close="yes") for item in dummies_data
-    ]
-    upload_data_to_commcare(
-        close_data,
-        project_slug,
-        "patient",
-        "external_id",
-        cc_user_name,
-        cc_api_key,
-        search_field="external_id",
-        create_new_cases="off",
-    )
-    # retrieve the dummies by contact id so we can get their case_ids, which we
-    # will attach to contacts we later upload
```

Comment:
Curious that this comment was removed since it seems to go with the code below which was not removed?

Reply:
And *that's* why there's code review. Thanks!